com.unity.render-pipelines.high-definition/Documentation~/master-stack-hair.md

```diff
@@ -154,5 +156,6 @@ Depending on the [Graph Settings](#graph-settings) you use, Shader Graph can add
 [!include[](snippets/shader-properties/advanced-options/override-baked-gi.md)]
 [!include[](snippets/shader-properties/advanced-options/support-lod-crossfade.md)]
 [!include[](snippets/shader-properties/advanced-options/add-precomputed-velocity.md)]
-[!include[](snippets/shader-properties/advanced-options/use-light-facing-normal.md)]
+[!include[](snippets/shader-properties/advanced-options/geometry-type.md)]
+[!include[](snippets/shader-properties/advanced-options/allow-radial-smoothness.md)]
```

Comment:
These [!includes[] look good, but I'd encourage you to test them using the Docs Tools package to make sure they work.